app/src/main/java/org/mozilla/fenix/tabhistory/TabHistoryViewHolder.kt

```diff
@@ -28,15 +29,17 @@ class TabHistoryViewHolder(
 
         history_layout.displayAs(LibrarySiteItemView.ItemType.SITE)
         history_layout.overflowView.isVisible = false
+        history_layout.titleView.text = item.title
         history_layout.urlView.text = item.url
         history_layout.loadFavicon(item.url)
 
-        history_layout.titleView.text = if (item.isSelected) {
-            buildSpannedString {
-                bold { append(item.title) }
-            }
+        val color = if (item.isSelected) {
+            ThemeManager.resolveAttribute(R.attr.tabHistoryItemSelectedBackground, history_layout.context)
         } else {
-            item.title
+            android.R.color.transparent
         }
+        history_layout.setBackgroundColor(
+            ContextCompat.getColor(history_layout.context, color)
```

Comment:
We have a helper for getting colors from attributes. You can also set a null drawable to clear the background.

```kt
if (item.isSelected) {
  history_layout.setBackgroundColor(context.getColorFromAttr(R.attr...)
} else {
  history_layout.background = null
}
```